app/models/complete_moab.rb

```diff
@@ -29,7 +31,7 @@ class CompleteMoab < ApplicationRecord
   #  assign the other complete_moab to preserved_objects_primary_moab
   has_one :preserved_objects_primary_moab, dependent: :destroy
 
-  validates :moab_storage_root, :preserved_object, :status, :version, presence: true
+  validates :status, :version, presence: true
```

Comment:
this is the only spot where I changed the explicit validations in the code, and I added the comment above "belongs_to" for good measure.